src/solutions/forms.py

```diff
@@ -47,7 +47,8 @@ def clean_file(self):
                     zip = zipfile.ZipFile(data)
                     if zip.testzip():
                         raise forms.ValidationError(_('The zip file seems to be corrupt.'))
-                    if sum(fileinfo.file_size for fileinfo in zip.infolist()) > 1000000:
+                    if sum(fileinfo.file_size for fileinfo in zip.infolist()) > max_file_size:
```

Comment:
better fix is:
`if sum(fileinfo.file_size for fileinfo in zip.infolist()) > (max_file_size * len(zip.infolist()): `